lms/resources/_js_config/__init__.py

```diff
@@ -143,6 +142,7 @@ def enable_oauth2_redirect_error_mode(
                 "OAuth2RedirectError": {
                     "authUrl": auth_url,
                     "invalidScope": is_scope_invalid,
+                    "errorCode": error_code,
```

Comment:
I agree that folding `invalidScope` into a new `errorCode` value in a follow-up PR would be a good thing to do.